### ISSUE TITLE
Showing "app" in menu other language

### ISSUE THREAD

**Issue by buttercrab** [lwouis/alt-tab-macos#228]
![image](https://user-images.githubusercontent.com/34997549/78867231-bbe2f200-7a7b-11ea-9b60-89ecc1ffa7b9.png)
This image is the screenshot of menu. It is in Korean and there is "app" at the back of second and fourth one. When I change to English, it won't appear.

**Comment by lwouis** [comment]
Wow that's not good! Yesterday I went to poeditor.com and also all the `.lproj` folders, and fixed all the `...` that were incorrect (correct form is `…`) or that were missing (on macOS, adding `…` after an item from the menubar means that it will not do the action immediately, but first prompt you for some more options).

During that process, I think I copy pasted wrong in 2 locations in Korean and 1 in Chinese. I fixed that and will release soon!

This process was very manual that's why I made a mistake. I wish Weblate would come back to me and that I could try and migrate away from poeditor. It's really hard to maintain sync between that site and the source code here on github 😅
